cluster/addons/storage-class/gce/windows/windows.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: standard-ntfs 
```

Comment:
I thought we agreed that ntfs StorageClass would not be installed out of the box
So for that PR it should be installed by test
Not by cluster kube-up?

Reply:
@msau42 clarified this is only installed by config-test. It is not installed by normal kube-up unless  `ENABLE_WINDOWS_STORAGE_CLASS` is explicitly set to true.